ee/docker-ee/ubuntu.md

```diff
@@ -23,73 +23,58 @@ Docker Engine - Community users should go to
 [Get Docker Engine - Community for Ubuntu](/install/linux/docker-ce/ubuntu.md)
 **instead of this topic**.
 
-To install Docker Engine - Enterprise, you need to know the Docker Engine -
-Enterprise repository URL associated with your trial or subscription. These
+To install Docker Engine - Enterprise, you need to know the Docker Engine - Enterprise repository URL associated with your trial or subscription. These
 instructions work for Docker on Ubuntu and for Docker on Linux, which includes
 access to Docker Engine - Enterprise for all Linux distributions. To get this
 information, do the following:
 
```

Comment:
Edit the paragraph to read: "Change the first sentence to read:

"The product URL associated with your trial or subscription is required to install Docker Engine - Enterprise, which can be attained using the following procedure (note that these instructions apply to Docker on Ubuntu and for Docker on Linux, which includes access to Docker Engine - Enterprise for all Linux distributions):"